include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -837,18 +901,19 @@ struct __gen_count_mask
     _GenMask __gen_mask;
 };
 
-template <typename _GenMask>
+template <typename _GenMask, typename _RangeTransform = oneapi::dpl::__internal::__no_op>
 struct __gen_expand_count_mask
 {
     template <typename _InRng, typename _SizeType>
     auto
     operator()(_InRng&& __in_rng, _SizeType __id) const
     {
+        auto __transformed_input = _RangeTransform{}(__in_rng);
```

Comment:
Similar comment to above with regards to adding `_RangeTransform` as a member.

Reply:
fixed, thanks.